### PR TITLE
Add adjoint(::Missing) method

### PR DESCRIPTION
**Pull request by quinnj**

### CODE REVIEW

**Review by andyferris — APPROVED** [review]

**Review by Sacha0 — APPROVED** [review]

**Review by nalimilan — APPROVED** [review]

**Review by fredrikekre — APPROVED** [review]
Test?

File: base/missing.jl [hunk 1]
```diff
@@ -65,7 +65,7 @@ for f in (:(!), :(+), :(-), :(identity), :(zero), :(one), :(oneunit),
           :(log2), :(exponent), :(sqrt), :(gamma), :(lgamma),
           :(iseven), :(ispow2), :(isfinite), :(isinf), :(isodd),
           :(isinteger), :(isreal), :(isnan), :(isempty),
-          :(iszero), :(transpose), :(float))
+          :(iszero), :(transpose), :(adjoint), :(float))
     @eval Math.$(f)(::Missing) = missing
```
Comment (ararslan) [inline base/missing.jl]: `adjoint` isn't defined in `Base.Math`.

**Comment by JeffBezanson** [comment]
Bump.

**Comment by nalimilan** [comment]
Re-bump.

**Comment by ararslan** [comment]
I hope you don't mind, Jacob, I rebased your branch against master and added tests.

**Comment by quinnj** [comment]
of course not :)

**Pull request merged**